pkg/elasticsearch/client.go

```diff
@@ -282,12 +282,12 @@ func readSAToken(tokenFile string) (string, bool) {
 	token, err := ioutil.ReadFile(tokenFile)
 
 	if err != nil {
-		logrus.Errorf("Unable to read auth token from file [%s]: %v", tokenFile, err)
+		log.Error(err, "Unable to read auth token from file", "file", tokenFile)
 		return "", false
 	}
 
 	if len(token) == 0 {
-		logrus.Errorf("Unable to read auth token from file [%s]: empty token", tokenFile)
+		log.Error(nil, "Unable to read auth token from file", "file", tokenFile)
```

Comment:
i haven't gotten to the log package yet, but what is the convention behind 
`log.Error(nil,...)` vs `log.Error(err...)` is it simply the cause?

Reply:
The first parameter is effectively a shortcut to `log.Error("message", "error", err)`, however if there is no specific `err` available then you can just pass nil which will translate to JSON `{"error": null}` which actually just removes the key from the log message altogether.